homeassistant/components/sensor/nmbs.py

```diff
@@ -0,0 +1,207 @@
+"""
+Get ride details and liveboard details for NMBS (Belgian railway).
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.nmbs/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, ATTR_ATTRIBUTION
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+import homeassistant.util.dt as dt_util
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'NMBS'
+DEFAULT_NAME_LIVE = "NMBS Live"
+
+DEFAULT_ICON = "mdi:train"
+DEFAULT_ICON_ALERT = "mdi:alert-octagon"
+
+CONF_STATION_FROM = 'station_from'
+CONF_STATION_TO = 'station_to'
+CONF_STATION_LIVE = 'station_live'
+
+REQUIREMENTS = ["pyrail==0.0.3"]
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_STATION_FROM): cv.string,
+    vol.Required(CONF_STATION_TO): cv.string,
+    vol.Optional(CONF_STATION_LIVE, default=None): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+})
+
+
+def get_time_until(departure_time=None):
+    """Calculate the time between now and a train's departure time."""
+    if departure_time is None:
+        return 0
+
+    delta = dt_util.utc_from_timestamp(int(departure_time)) - dt_util.now()
+    return round((delta.total_seconds() / 60))
+
+
+def convert_ms_to_sec(delay=0):
```

Comment:
Is it milliseconds or seconds that were converting?

Reply:
Milliseconds to seconds, as the function descriptor says.

Reply:
The users of this function seem to expect minute as return value type. We're also dividing by 60, which suggests we're trying to go from seconds to minutes. Am I missing something?

Reply:
You are right, sorry. I seemed to have made this function before I used the time module to get the total seconds before sending it off to this function.